not RBAC friendly?

I'm using kubernetes 1.7.1, and have a following problem:

```
time="2017-07-26T15:36:02Z" level=info msg="elasticsearch operator starting up!"
time="2017-07-26T15:36:02Z" level=info msg="Using Variables:"
time="2017-07-26T15:36:02Z" level=info msg="   baseImage: upmcenterprises/docker-elasticsearch-kubernetes:5.3.1"
time="2017-07-26T15:36:02Z" level=info msg="Using InCluster k8s config"
panic: User "system:serviceaccount:default:elasticsearch-operator" cannot get thirdpartyresources.extensions at the cluster scope. (get thirdpartyresources.extensions elasticsearch-cluster.enterprises.upmc.com)
goroutine 1 [running]:
github.com/upmc-enterprises/elasticsearch-operator/pkg/k8sutil.(*K8sutil).CreateKubernetesThirdPartyResource(0xc420186b10, 0x10, 0xc420157fe0)
	/Users/slokas/godev/src/github.com/upmc-enterprises/elasticsearch-operator/pkg/k8sutil/k8sutil.go:285 +0x1b2
github.com/upmc-enterprises/elasticsearch-operator/pkg/controller.(*Controller).init(0xc420523e70, 0x7f69c66044b0, 0x0)
	/Users/slokas/godev/src/github.com/upmc-enterprises/elasticsearch-operator/pkg/controller/controller.go:69 +0x2e
github.com/upmc-enterprises/elasticsearch-operator/pkg/controller.(*Controller).Run(0xc420523e70, 0xc420157fe0, 0xc4203f6960)
	/Users/slokas/godev/src/github.com/upmc-enterprises/elasticsearch-operator/pkg/controller/controller.go:58 +0x2f
main.Main(0xc4200001a0)
	/Users/slokas/godev/src/github.com/upmc-enterprises/elasticsearch-operator/cmd/operator/main.go:85 +0x340
main.main()
	/Users/slokas/godev/src/github.com/upmc-enterprises/elasticsearch-operator/cmd/operator/main.go:110 +0x22
```

Does that mean that RBAC rules are missing?

--- a/README.md
+++ b/README.md
@@ -85,8 +85,12 @@ spec:
 To deploy the operator simply deploy to your cluster:
 
 ```bash
-$ kubectl create -f https://raw.githubusercontent.com/upmc-enterprises/elasticsearch-operator/master/example/controller.yaml
+$ kubectl create ns operator
+$ kubectl create -f https://raw.githubusercontent.com/upmc-enterprises/elasticsearch-operator/master/example/controller.yaml -n operator
 ```
+
+_NOTE: In the example we're putting the operator into the namespace `operator`. If you want to change this, then make sure to update the RBAC rules in the [example/controller.yaml](example/controller.yaml
+ ) spec to match the namespace desired._
 
 # Create Example ElasticSearch Cluster
 

--- a/example/controller.yaml
+++ b/example/controller.yaml
@@ -3,20 +3,26 @@ kind: ServiceAccount
 metadata:
   name: elasticsearch-operator
 ---
-apiVersion: rbac.authorization.k8s.io/v1alpha1
+apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
   name: elasticsearch-operator
 rules:
 - apiGroups: ["extensions"]
-  resources: ["deployments", "thirdpartyresources"]
-  verbs: ["create", "get", "update", "delete"]
+  resources: ["deployments", "thirdpartyresources", "replicasets"]
+  verbs: ["create", "get", "update", "delete", "list"]
 - apiGroups: ["storage.k8s.io"]
   resources: ["storageclasses"]
-  verbs: ["get", "list", "create", "delete"]
+  verbs: ["get", "list", "create", "delete", "deletecollection"]
 - apiGroups: [""]
-  resources: ["services"]
+  resources: ["services", "secrets"]
   verbs: ["create", "get", "update", "delete"]
+- apiGroups: ["batch"]
+  resources: ["cronjobs", "jobs"]
+  verbs: ["create", "get", "deletecollection", "delete"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["list", "get", "watch"]
 - apiGroups: ["apps"]
   resources: ["statefulsets"]
   verbs: ["*"]
@@ -24,10 +30,11 @@ rules:
   resources: ["elasticsearchclusters"]
   verbs: ["*"]
 ---
-apiVersion: rbac.authorization.k8s.io/v1alpha1
-kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
 metadata:
   name: elasticsearch-operator
+  namespace: operator
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
@@ -35,11 +42,13 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: elasticsearch-operator
+  namespace: operator
 ---
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: elasticsearch-operator
+  namespace: operator
 spec:
   replicas: 1
   template:

